README.md

```diff
@@ -138,10 +138,14 @@ Starting with v0.10.0, commands with table output (i.e. `handlr list` and `handl
 sudo pacman -S handlr-regex
 ```
 
-Optionally you can also install `xdg-utils-handlr` from the AUR to replace `xdg-open`:
+Optionally you can also use `handlr-regex` as a replacement for `xdg-open` by putting a shadowing script into `$HOME/.local/bin` directtory (or any other `$PATH` directory). Use the following shell snippet as an example:
 
 ```sh
-paru -S xdg-utils-handlr
+echo "#!/bin/sh
```

Comment:
Could you change this into the script itself? I think that would be clearer and more general.

Reply:
Sure, done